### PR TITLE
[MIG] l10n_fr_naf_ape : Migrated to 10.0

### PR DESCRIPTION
**Pull request by mayjo79**
Just after  l10n_eu_nace, I migrated l10n_fr_naf_ape

### CODE REVIEW

**Comment by mayjo79** [comment]
Could someone explain to me what is the problem in runbot check ?
Thanks

**Comment by mayjo79** [comment]
Hello @pedrobaeza  
Could you explain to me what is the problem in runbot check ?
Thanks

**Comment by pedrobaeza** [comment]
You have to update https://github.com/OCA/l10n-france/blob/10.0/.travis.yml to include in some lines TESTS="1", as it's disabled in all lines. Check template: https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L57

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +29,8 @@ env:
   - TRANSIFEX="1"
   - ODOO_REPO="odoo/odoo" LINT_CHECK="0"
   - ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - TESTS="1" ODOO_REPO="odoo/odoo"
```
Comment (pedrobaeza) [inline .travis.yml]: You should replace the lines, not add them, or the CIs time will increase.

**Comment by mayjo79** [comment]
Hello @pedrobaeza
What is missing to merge this pull request ?

**Comment by pedrobaeza** [comment]
Hi, @mayjo79, you need to find reviewers for this. I'm not the ideal one, as I don't know about French specifics. Maybe @alexis-via can help. You can also review other PRs (from this repo or something related) and ask to review yours in return.

**Comment by mayjo79** [comment]
Hello @clonedagain 
As you are the author of this module, could you review this PR ?
Thanks

**Review by gurneyalex — APPROVED** [review]

**Review by clonedagain — APPROVED** [review]
I didn't have the opportunity to test it on runbot but it should be fine.

**Review by alexis-via — APPROVED** [review]

**Pull request merged**